azkaban-execserver/src/main/java/azkaban/execapp/ServerStatisticsServlet.java

```diff
@@ -91,30 +91,70 @@ protected void fillRemainingMemoryPercent(ExecutorInfo stats){
       }
 
       long totalMemory = 0;
-      long  freeMemory = 0;
+      long totalFreeMemory = 0;
+      long parsedResult = 0;
 
       // process the output from bash call.
       // we expect the result from the bash call to be something like following -
       // MemTotal:       65894264 kB
-      // MemFree:        61104076 kB
-      if (output.size() == 2) {
+      // MemFree:        57753844 kB
```

Comment:
Let's add a TODO so we remember to create a utils method and remove redundancy later on. This method method perform almost same thing and in similar manner as https://github.com/azkaban/azkaban/blob/master/azkaban-common/src/main/java/azkaban/utils/SystemMemoryInfo.java#L105